go/prometheusexporter/collector/collector.go

```diff
@@ -893,19 +945,46 @@ func (c *Collector) recordAjaxMetrics(m *metric) {
 	c.actionRegistry <- action
 }
 
+func (c *Collector) recordWebVitals(m *metric) {
+	c.mutex.RLock()
+	defer c.mutex.RUnlock()
+	p := m.props
+	action := p["action"]
+
+	labels := make(map[string]string)
+	labels["app"] = p["app"]
+	labels["env"] = p["env"]
+	labels["action"] = action
+
+	for _, vital := range m.webvitals {
+		if vital.FID != nil {
+			c.actionMetrics.webvitalsFid.With(labels).Observe(*vital.FID)
+		}
+		if vital.CLS != nil {
+			c.actionMetrics.webvitalsCls.With(labels).Observe(*vital.CLS)
+		}
+		if vital.LCP != nil {
+			c.actionMetrics.webvitalsLcp.With(labels).Observe(*vital.LCP)
```

Comment:
LCP is a value measured in milliseconds: https://github.com/skaes/logjam-tools/blob/add-web-vitals-endpoint/go/formats/webvitals/webvitals.go#L26-L27

We'll need to convert it to seconds.